dd-java-agent/instrumentation/weaver/build.gradle

```
@@ -3,23 +3,23 @@ apply plugin: 'scala'
 
 muzzle {
```

We want to also perform muzzle checks in the previous org, to ensure that changes to the instrumentation don't break on those versions too.
```suggestion
muzzle {
  pass {
    group = 'com.disneystreaming'
    module = 'weaver-cats_3'
    versions = '[0.8.4,)'
  }
```